### DISCUSSION TITLE
Schedules cheat sheet to crontab guru

### DISCUSSION THREAD

**Discussion by aussieserverhosts** [pelican-dev/panel#497] [Ideas]
Instead of just a table of a cheat sheet
can you essentially implement what would essentially be crontab.guru in the client facing Schedules

**Comment by QuintenQVD0** [comment]
If you create a cron it will tell you when the next run is after you made it so it gives a basic idea if you did it right?

**Comment by rmartinoscar** [comment]
![image](https://github.com/user-attachments/assets/edf60c85-d4f4-40e8-9bbe-865b0d012612)
![image](https://github.com/user-attachments/assets/54eab7f1-d1e1-4dbc-bde3-0127a35d4a7f)
You know have those buttons to help you or you can set it manually
